asssignments/rajesh/assignment-3/Rajeev/main.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import "fmt"
+
+func printFibonacciSeries(num int) {
+	a := 0
+	b := 1
+	c := b
```

Comment:
```suggestion
	var int c
```
You can just declare like this here